Content/posts/launching-language-and-platform-package-compatibility.md

```diff
@@ -1,7 +1,7 @@
 ---
 date: 2020-08-20 12:00
 title: Launching Language and Platform Package Compatibility
-description:
+description: We decided that one of the most important pieces of information we could provide on a package metadata page was what versions of Swift, and what platforms it was compatible with. BuildingtThat feature turned out to be quite an epic journey.
```

Comment:
Just noticed a typo after merging: `BuildingtThat` - unless it's a display issue in Github on iOS. Can't check right now, I'm afraid.

Reply:
Yep, you're right! Making another PR.